services/core-web/src/components/noticeOfWork/applications/process/ProcessPermit.js

```diff
@@ -214,7 +334,15 @@ export class ProcessPermit extends Component {
     this.props.generateNoticeOfWorkApplicationDocument(
       documentTypeCode,
       payload,
-      "Successfully created document and attached it to Notice of Work"
+      "Successfully created document and attached it to Notice of Work",
+      () => {
+        if (
+          documentType.now_application_document_type_code === "PMA" ||
+          documentType.now_application_document_type_code === "PMT"
+        ) {
+          afterSuccess(values, "Permit has been successfully issued for this application.", "AIA");
+        }
```

Comment:
can you use the const approvedCode for every instance of "AIA"

Reply:
Done